modules/nf-core/umitools/dedup/main.nf

```diff
@@ -1,11 +1,13 @@
 process UMITOOLS_DEDUP {
     tag "$meta.id"
-    label "process_medium"
+    label "process_single"
+    label "process_long"
```

Comment:
Do we need to give this process_long, if it has fixed the transcriptome time issue?